packages/desktop-client/src/components/accounts/Reconcile.tsx

```diff
@@ -66,23 +80,10 @@ export function ReconcilingMessage({
           <View style={{ color: theme.tableText }}>
             <Text style={{ fontStyle: 'italic', textAlign: 'center' }}>
               <Trans>
-                Your cleared balance{' '}
-                <strong>
-                  {{ clearedBalance: format(cleared, 'financial') }}
-                </strong>{' '}
-                needs{' '}
-                <strong>
-                  {{
-                    difference:
-                      (targetDiff > 0 ? '+' : '') +
-                      format(targetDiff, 'financial'),
-                  }}
-                </strong>{' '}
-                to match
+                Your cleared balance <strong>{clearedBalance}</strong> needs{' '}
```

Comment:
I'm not entirely sure if this new approach will work with the translations. Please let me know if there is a different way to approach this.

The problem: TS was complaining about the previous implementation.